.github/workflows/ci.yml

```diff
@@ -53,8 +53,10 @@ jobs:
         with:
           go-version-file: 'go.mod'
           cache: true
-      - name: Run linter
```

Comment:
Sorry for going back and forth. I meant we could have a new job in the same yaml.
https://github.com/containerd/nerdctl/blob/main/.github/workflows/test.yml is a good example with lint, project test-unit being different jobs.